core-bundle/contao/themes/flexible/styles/basic.css

```diff
@@ -681,6 +729,11 @@ select[multiple] {
 	margin: 0 0 6px 21px !important;
 }
 
+.tl_checkbox_container .tl_checkbox {
+	margin-top: 1px;
+	margin-bottom: 1px;
+}
+
```

Comment:
What does this do?

Reply:
I guess you got confused about this since this CSS was below the "BC comment".
I fixed the order in the latest commit.

Reply:
I also think that we can remove the two `.tl_checkbox_container .tl_checkbox` rules (one defining a `1px` margin and the other one removing it again).

Reply:
> I also think that we can remove the two `.tl_checkbox_container .tl_checkbox` rules (one defining a `1px` margin and the other one removing it again).

Removed in 0ad58fbc113964e55fa52de25d7e6666bdd2d43a